core/lib/default_da_clients/src/lib.rs

```diff
@@ -0,0 +1,3 @@
+pub mod object_store;
+
```

Comment:
Nit: This newline looks unnecessary; e.g., it prevents `rustfmt` from sorting modules in the alphabetical order.